docs/joss_paper/paper.md

```diff
@@ -56,9 +56,9 @@ bibliography: paper.bib
 
-`DeepReg` is a Python package that implements a class of image registration algorithms based on deep neural networks. Both unsupervised algorithms [@de2019deep] and weakly-supervised algortihms [@hu2018label] have been implemented. In addition, combining unsupervised losses, such as those derived from image similarity measures, and weak supervision, such as overlap between corresponding anatomical and pathological regions, is at the core of this framework. In this paper, we outline the algorithms and several representative clinical applications, available as 'demos' at the time of writing. A summary of data input and output is also provided to describe the implemented dataset loaders and training data sampling methods. Collectively, we demonstrate the versatility of the package, by providing a wide range of clinical applications as example demonstrations in the `demos` folder.
+`DeepReg` is a Python package that implements a class of image registration algorithms based on deep neural networks. Both unsupervised algorithms [@de2019deep] and weakly-supervised algortihms [@hu2018label] have been re-implemented. In addition, combining unsupervised losses, such as those derived from image similarity measures, and weak supervision, such as overlap between corresponding anatomical and pathological regions, is at the core of this framework. In this paper, we first briefly describe the major algorithms in DeepReg. A summary of the implementatoin of these algorithms is then provided. The model input and output, with- and without labels, the data sampling methods and the six dataset loaders implementing these are also described. Finally, we demonstrate the versatility of the package by providing a wide range of clinical applications as _DeepReg Demos_ in the `demos` folder, which are also briefly discussed in this paper.
 
-`DeepReg` provides a command line tool option that implements basic functionalities including model training, prediction and image warping, with options and configurations being specified through the command line arguments and/or configuration files in .yaml format.
+`DeepReg` provides command line tool options that implement basic functionalities including model training, prediction and image warping. Options and configurations being specified through the command line arguments and/or configuration files in yaml format. This design choice has been motivated by both research and education purposes orginally set out in the DeepReg project, which aims to provide benchmarking implementations of standard deep-learning-based image registration algorithms. These implementations, together with their documentation, tutorials and demos, simplify workflows for developping novel methodology, rapid utilising latest development and accessing quality research topics in this area.
```

Comment:
"Options and configurations being specified through the command line arguments and/or configuration files in yaml format." This reads awkward to me.

"yaml files and command line arguments are used as the basis of configuring the various training options of DNNs with DeepReg."

Reply:
"which aims to provide benchmarking implementations" -> are there any benchmarks? Alternatively, "which aims to provide access to fast prototyping of standard algorithms in deep-learning-based image reg algorithms."

Reply:
"The repository of ready-built DNN architectures, together with their documentation, tutorials and demos, simplify workflow to both new academics in the area and for those developing novel methodology,"

Reply:
> "which aims to provide benchmarking implementations" -> are there any benchmarks? Alternatively, "which aims to provide access to fast prototyping of standard algorithms in deep-learning-based image reg algorithms."

DeepReg is the bench! :D